server/game/cards/03_TWI/units/VanguardDroidBomber.ts

```diff
@@ -0,0 +1,25 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { Trait } from '../../../core/Constants';
+
+export default class VanguardDroidBomber extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '3410014206',
+            internalName: 'vanguard-droid-bomber'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'If you control another Separatist unit, deal 2 damage to an enemy base.',
```

Comment:
For our ability titles we don't include "if" clauses, since the ability will only appear at all if the condition is already met. So in this case it can just be `Deal 2 damage to an enemy base`

Reply:
I am yet very fluent on what I should put in the title. I learnt from last time that "when played" was not required in the title.
Now I understand it's also the case for the condition.
Will do my best for the next one :)